### PR TITLE
MM-17445 add test for "Words that trigger mentions support Chinese"

### PR DESCRIPTION
**Pull request by nadalfederer**
Summary
Add test "Words that trigger mentions support Chinese"

Test steps:

1. Post message with 番茄 in it from user-1 on a channel 
2. Check that this message highlighted and initial user is mentioned
3. Post message with 我爱吃番茄炒饭 in it from sysadmin on an another channel
4. Check that this message highlighted and the initial user is mentioned

Ticket Link
Jira ticket [MM-17445](https://mattermost.atlassian.net/browse/MM-17445)
closes https://github.com/mattermost/mattermost-server/issues/11745

### CODE REVIEW

**Comment by saturninoabril** [comment]
Hello @nadalfederer, will delay review to give way for the ongoing release period.

File: e2e/cypress/integration/at_mentions/at_mentions_spec.js [hunk 1]
```diff
@@ -224,4 +230,76 @@ describe('at-mention', () => {
             cy.get('#sidebarItem_saepe-5').click({force: true});
```
Comment (saturninoabril) [inline e2e/cypress/integration/at_mentions/at_mentions_spec.js]: When all tests of this spec file are run, the newly added test is failing due to above line.  Please delete.
```suggestion
```

Comment (nadalfederer) [reply e2e/cypress/integration/at_mentions/at_mentions_spec.js]: I fixed it

**Pull request merged**